analyzer.go

```diff
@@ -1,22 +1,20 @@
 package lifecycle
 
 import (
-	"fmt"
-	"path/filepath"
-
 	"github.com/buildpacks/imgutil"
 	"github.com/pkg/errors"
 
+	"github.com/buildpacks/lifecycle/api"
 	"github.com/buildpacks/lifecycle/buildpack"
-	"github.com/buildpacks/lifecycle/launch"
 	"github.com/buildpacks/lifecycle/platform"
 )
 
 type Analyzer struct {
-	Buildpacks []buildpack.GroupBuildpack
-	LayersDir  string
-	Logger     Logger
-	SkipLayers bool
+	Buildpacks  []buildpack.GroupBuildpack
+	LayersDir   string
+	Logger      Logger
+	SkipLayers  bool
```

Comment:
Maybe we should move this into a separate grouping and add a doc comment that makes it clear, this is only for older APIs?